framework/test/unit/modules/interoperability/store.spec.ts

```diff
@@ -224,9 +225,17 @@ describe('Base interoperability store', () => {
 			nonce: BigInt('0'),
 		};
 
+		const createTerminatedStateAccountContext = {
+			eventQueue: new EventQueue(0),
+		};
+
 		it('should set appropriate terminated state for chain id in the terminatedState sub store if chain account exists for the id and state root is provided', async () => {
```

Comment:
I think we need to also assert the emitted event `EVENT_NAME_CHAIN_ACCOUNT_UPDATED ` and also for `await terminatedStateSubstore.set` and event  `EVENT_NAME_TERMINATED_STATE_CREATED`